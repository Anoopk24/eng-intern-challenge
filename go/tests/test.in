A123
Hello world
42
O..OO.
.O.OOOO..OO.
.....OO.....O.O...OO...........O.OOOO.....O.O...OO....
