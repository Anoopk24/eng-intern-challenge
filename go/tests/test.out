.....OO......O.OOOO.....O.O...OO....
.....OO.OO..O..O..O.O.O.O.O.O.O..OO........OOO.OO..OO.O.OOO.O.O.O.OO.O..
.O.OOOOO.O..O.O...
o
>
Abc 123
